src/app/components/pages/IsaacCompetition/IsaacCompetition.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect } from "react";
+import { SITE_SUBJECT_TITLE } from "../../../services";
+import { TitleAndBreadcrumb } from "../../elements/TitleAndBreadcrumb";
+import { Button, Col, Container, Row } from "reactstrap";
+import { Link } from "react-router-dom";
+import content from "./content";
+
+const { section1 } = content;
+
+export const IsaacCompetition = () => {
+  useEffect(() => {
+    document.title = "Isaac " + SITE_SUBJECT_TITLE;
+  }, []);
+
+  const CompetitionButton = () => {
+    const competitionInterestButton = [{ to: "/competition-sign-up", label: "Express your interest" }];
+
+    return (
+      <Row>
+        {competitionInterestButton.map(({ to, label }) => (
+          <Col xs={12} className="py-1" key={to}>
+            <Button size="lg" tag={Link} to={to} block className="primary-button text-light">
+              {label}
+            </Button>
+          </Col>
+        ))}
+      </Row>
+    );
+  };
+
+  return (
+    <>
+      <Container>
+        <TitleAndBreadcrumb currentPageTitle="Isaac Competition" breadcrumbTitleOverride="Isaac Competition" />
+      </Container>
+      <div id="competition-headline-section">
+        <Container className="pt-4 pb-4 z1">
+          <Row>
+            <h1 className="primary-heading pl-3">National Computer Science Competition</h1>
+            <Col xs={12} md={6} lg={8} className="pb-3">
+              <p className="mt-4 body-text">{section1.header.section}</p>
+              <p className="mt-4 mb-0 body-text">
+                <span style={{ fontWeight: 700 }}>{section1.note.text.split(":")[0]}:</span>{" "}
+                {section1.note.text.split(":")[1].split("X")[0]}
```

Comment:
i see what you've done here, I'd split up the actual content in this case in the new file, rather than doing it programmatically doing it. "Text Part 1" and "Text Part 2" with the links being added like you have below :)